R/erf_dgumbel.R

```diff
@@ -0,0 +1,6 @@
+erf_dgumbel <- function(x, mu=0, sigma=1){
```

Comment:
Tem algum bom motivo para os argumentos estarem fixados? no `pgumbel` não está
```suggestion
erf_dgumbel <- function(x, mu = 0, sigma = 1){
```

Reply:
Estava testando o `integrate`, mas posso utilizar como argumentos da `integrate` mesmo